### DISCUSSION TITLE
Redirect after email verification

### DISCUSSION THREAD

**Discussion by Electric-Dragon** [supabase/supabase#7076] [Questions]
Hello,
I want to redirect after email verification
<img width="1129" alt="Bildschirmfoto 2022-05-31 um 1 32 31 PM" src="https://user-images.githubusercontent.com/45729391/171123741-095303c4-a9ab-4c00-a639-05da23d75479.png">

What do i keep the configuration as? I want to redirect to localhost and the main website depending on where the request originated from

**Comment by Gerald-Gmainer** [comment]
You can specify the redirectTo link 
```
const redirectTo = "http://localhost:8080/"
const { data, error } = await supabase.auth.api.inviteUserByEmail(email, { redirectTo });
```

The redirectTo links must be set in the "Additional redirect URLs" (comma separated)

I tried it with `inviteUserByEmail`, should also work with magic links

**Reply by Electric-Dragon** [reply]
@Gerald-Gmainer what about `generateLink()` with type invite

**Reply by Gerald-Gmainer** [reply]
https://supabase.com/docs/reference/javascript/auth-api-generatelink
